plugins/dashboard/conflicts_livefeed.go

```diff
@@ -226,3 +218,105 @@ func issuerOfOldestAttachment(branchID ledgerstate.BranchID) (id identity.ID) {
 	})
 	return
 }
+
+type timeHeapElement struct {
+	conflictID  ledgerstate.ConflictID
+	updatedTime time.Time
+}
+
+type timeHeap []*timeHeapElement
+
+func (h timeHeap) Len() int {
+	return len(h)
+}
+
+func (h timeHeap) Less(i, j int) bool {
+	return h[i].updatedTime.Before(h[j].updatedTime)
+}
+
+func (h timeHeap) Swap(i, j int) {
+	h[i], h[j] = h[j], h[i]
+}
+
+func (h *timeHeap) Push(x interface{}) {
+	*h = append(*h, x.(*timeHeapElement))
+}
+
+func (h *timeHeap) Pop() interface{} {
+	n := len(*h)
+	data := (*h)[n-1]
+	(*h)[n-1] = nil
+	*h = (*h)[:n-1]
+	return data
+}
+
+var _ heap.Interface = &timeHeap{}
+
+type boundedConflictMap struct {
+	conflicts    map[ledgerstate.ConflictID]*conflict
+	branches     map[ledgerstate.BranchID]*branch
+	conflictHeap *timeHeap
+}
+
+func (b *boundedConflictMap) conflict(conflictID ledgerstate.ConflictID) (conflict *conflict, exists bool) {
+	conflict, exists = b.conflicts[conflictID]
```

Comment:
```suggestion
	return b.conflicts[conflictID]
```

Parameters should be clear bc of their naming.